tests/scale/results/summary.md

```diff
@@ -0,0 +1,139 @@
+See [graph](/tests/scale/results/1.0/TestScale_Listeners/CPU.png).
+
+**Memory**: Gradual increase in memory. Topped out at 40MiB.
+See [graph](/tests/scale/results/1.0/TestScale_Listeners/Memory.png).
+
+**Time To Ready**: Time to ready numbers consistently under 3s. 62nd Listener had longest TTR of 3.02s.
+See [graph](/tests/scale/results/1.0/TestScale_Listeners/TTR.png).
+
+### Scale HTTPS Listeners
+
+| Total Reloads | Total Reload Errors | Average Reload Time (ms) |
+|---------------|---------------------|--------------------------|
+| 130           | 0                   | 151.49397590361446ms     |
+
+**NGINX Errors**: None.
+
+**NGF Errors**: None.
+
+**Pod Restarts**: None.
+
+**CPU**: Steep linear increase as NGF processed all the Services and Secrets. Dropped off during scaling of Listeners.
+See [graph](/tests/scale/results/1.0/TestScale_HTTPSListeners/CPU.png).
+
+**Memory**: Mostly linear increase. Topping out at right under 50MiB.
+See [graph](/tests/scale/results/1.0/TestScale_HTTPSListeners/Memory.png).
+
+**Time To Ready**: The time to ready numbers were pretty consistent (under 3 sec) except for one spike of 10s. I believe
+this spike was client-side because the NGF logs indicated that the reload successfully happened under 3s.
+See [graph](/tests/scale/results/1.0/TestScale_HTTPSListeners/TTR.png).
+
+### Scale HTTPRoutes
+
+| Total Reloads | Total Reload Errors | Average Reload Time (ms) |
+|---------------|---------------------|--------------------------|
+| 1001          | 0                   | 354.3878787878788ms      |
+
+**NGINX Errors**: None.
+
+**NGF Errors**: None.
+
+**Pod Restarts**: None.
+
+**CPU**: CPU mostly oscillated between .04 and .06. Several spikes over .06.
+See [graph](/tests/scale/results/1.0/TestScale_HTTPRoutes/CPU.png).
+
+**Memory**: Memory usage gradually increased from 25 - 150MiB over course of the test with some spikes reaching up to
+200MiB. See [graph](/tests/scale/results/1.0/TestScale_HTTPRoutes/Memory.png).
+
+**Time To Ready**: This time to ready graph is unique because there are three plotted lines:
+
+- Blue Line: 2-second delay after adding a new HTTPRoute.
+- Red Line: No delay after adding a new HTTPRoute.
+- Green Line: 10-second delay after adding a new HTTPRoute
+
+The Blue and Red lines are incomplete because the tests timed out. However, I think the implications are pretty clear.
+The more time that passes between scaling events, the smaller the time to ready values are. This is because NGF
+re-queues all the HTTPRoutes after updating their statuses. This is because the HTTPRoute has changed after we write its
+status. This is compounded by the fact that NGF writes status for every HTTPRoute in the graph on every configuration
+update. So if you add HTTPRoute 100, NGF will update the configuration with this new route and then update the status of
+all 100 HTTPRoutes in the graph.
+
+Related issues:
```

Comment:
I wonder if we need a dedicated bug here. 

if we take avg reload time into consideration, the lower bound for adding 1000 HTTPRoutes should be `(0.354 * 1000)/60 = 354 seconds`, which is 6 min.

The fact that the test needs 7 hours to run is somewhat disturbing.

Reply:
Good point about the lower bound. I was surprised by the test duration as well. I do think the root causes are theses two issues but I could be missing something. I'll add another issue for this and link the issues below.

Reply:
https://github.com/nginxinc/nginx-gateway-fabric/issues/1122